program/src/matching.rs

```diff
@@ -271,6 +271,16 @@ pub struct BookSideIter<'a> {
     right: usize,
 }
 
+pub struct Order {
+    // The quantity, in base_lot
+    pub quantity: i64,
+    // Marginal Price, the price to place the order at, in quote (per base_lot)
+    pub price: i64,
+    // The resulting total amount that will be spent, in quote_lot (without fees)
+    pub size: i64,
```

Comment:
size currently unused?

Reply:
Yes size is not used anymore, can be removed